packages/addons-manager/bundle/config/values.yaml

```diff
@@ -6,6 +6,7 @@ tanzuAddonsManager:
   namespace: tkg-system
   createNamespace: true
   deployment:
+    leaderElection: false
```

Comment:
Currently addons-manager is installed using tanzu-framework and tkg composite package.

i.e. Installing tkg package would install framework package and installing tanzu-framework would install addons-manager and other packages.

So this `leaderElection: false` also has to be added in framework and tkg package

i.e. Please add leaderElection: false under

https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/tkg/bundle/config/values.yaml#L39
https://github.com/vmware-tanzu/tanzu-framework/blob/main/packages/framework/bundle/config/values.yaml#L28

Reply:
Good to know! Thanks for pointing that out. PR has been updated.